release_notes.md

```diff
@@ -1,64 +1,22 @@
-
----
-
-#### Enhanced Features for Precision and Compliance
-- **Hunts Module Relocation**: The Hunts module is now part of Threat Intel Management, offering cohesive access to threat analysis tools.
-- **Mandatory Fields Compliance**: Feed modules now meet STIX/TAXII 2.0 requirements with key fields marked as mandatory.
-- **System View Template Update**: The System View Template (SVT) now includes new fields, ensuring comprehensive and up-to-date data representation.
-- **Feed Configuration Settings**: The sections of feed configuration settings have been rearranged under 3 tabs:
-    - *Feed Sources* - Find, Install, and Configure new feed connectors through Content Hub.
-    - *TAXII Server* - Configure a TAXII Server to generate an outgoing feed to share threat intelligence externally.
-    - *Threat Feed Rules* - Configure threat feed to manage threat intelligence from various sources.
-- The **Workspace** now carries a description field to better communicate the purpose of setting up a workspace
-Stay aligned with industry standards while managing your threat feeds seamlessly.
-
----
-
-#### Streamlined Threat Feed Management
-
-Managing threat data just got easier with these powerful updates:
-
-- **Unstructured Feeds Made Simple**: The new Import Unstructured Feeds button on the Threat Feed page lets you upload files and create feed records effortlessly.
-- **MITRE ATT&CK Connector with Default Configuration**: The MITRE ATT&CK connector now installs with a default configuration. You no longer need to manually configure it to ingest threat feeds from the MITRE organization.
-- **Threat Feed Rules**: New rules help streamline threat feed management:
-    - **Link Threat Feed to Indicators**: (Earlier *Feed to Indicator Linking*) Updates the indicator reputation as per the matching feed.
-    - **Ingest Unstructured Threat Feeds**: Ingests unstructured feeds from files or email attachments
-- Following are added for *MITRE&reg;* connector during installation:
-    - A default configuration
-    - Data ingestion playbooks
-    - An ingestion schedule
-- Following are removed for *Fortinet FortiGuard Threat Intelligence* during installation
-    - Data ingestion playbooks
-    - An ingestion schedule
-
-    These are created by the configuration wizard later. 
-
-- The configuration wizard creates the following datasets and triggers the *MITRE&reg;* integration's data ingestion:
-    - FortiGuard Outbreak Threat Feeds
-    - FortiGuard Threat Intel Feeds
-    - Phishing Threat Feeds
-
-These features enable seamless handling of diverse data formats and automated configurations, saving time and effort.
+#### Added support for ingesting FortiGuard Threat Reports
+Introduced support for ingesting FortiGuard Threat Reports! To enable this, the following updates are made to the `10 - SP - TIM Automation` playbook collection:
+- Added the following playbooks:
```

Comment:
This statement is not required here and Reports can be igested automatically when user runs configuration wizard. @cs-sujeet-ahirrao  please confirm